website/content/docs/developers/simulator-mode.md

```diff
@@ -39,8 +39,17 @@ simulator.deactivate("test");
 simulator.shutdown();
 ```
 
-To kill a topology, you could also simply terminate this process.
+To kill a topology, you could also terminate the process.
+
+The simulator mode will run in separate threads other than main thread. All those interfaces are thread-safe. This means that you could invoke these interfaces in other threads and monitor the corresponding behaviors interactively.
+
+# Debug topology using IntelliJ
+
+Bolts and Spouts run as separate threads in simulator. To add breakpoints inside a bolt/spout, the Suspend Policy of the breakpoint needs to be set as Thread. To change the Suspend Policy, Right click on the breakpoint as shown in the following image:
+
+![Set Breakpoint](/img/intellij-set-breakpoint.jpg)
+
+If output and logs are not convenient to be checked in intellij console, they can be saved to local file by choose `Run -> Edit Configurations....` and set as shown in the following image:
```

Comment:
Thanks for the clarification @lucperkins.

Do you think I need to revert the 'you' changes? Or we can just merge this PR here?